### DISCUSSION TITLE
Domains, Maps and Rafts: common terminology for discussions on inner development (aka ontogenenis or ontological evolution)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#914] [Research]
**🚩 UPDATE: published at https://lifeitself.org/blog/domains-maps-rafts**

Distilling the "domains, maps, rafts" terminology we're regularly using. These aren't new concepts at all, but they have various different names and having one agreed set of names is helpful.

## Definitions

🏰 **Domains (aka Lines)**: domains are a specific ontogenetic area ie. a specific area of human development e.g. cognitive complexity and morality could be two domains.

🗺️ **Maps**: domains have "maps" with different locations aka stages, levels, altitudes.

🧭 **Compasses**: we need to locate ourselves on the map. Compasses help us do that.

⛵ **Rafts**: things like practices or environments that help move between locations in a domain.

## Notes

- **Domains should be distinct** (even if co-evolving): domains should be in some sense distinct in the sense that one can develop in one domain independently of another, at least to some extent. Take cognitive complexity and morality. These could be two distinct domains because you could be a super smart psychopath -- i.e. you can develop in domain of cognitive complexity without necessarily evolving in morality. 
- **Maps can include sequencing (or not)**: include both locations and how the locations relate to each other. For example, perhaps there is a *sequence* where one location one follows another and even how that unfolding or development occurs in a general sense (e.g. piaget's stages are a map). At the same time, sequences are not required. Maybe there is no specific order in which you visit locations or it is only a probable sequencing. For example, Martin emphasizes that you land in any one of locations 1-4 when entering fundamental wellbeing and that you don't need to move linearly through the locations -- in fact it is normal to jump around and even to move from higher locations to lower locations over time.
- **Locations (vs stages)**: we prefer the term location as it has a more neutral sense that stages (which suggest hierarchy etc). Furthermore, locations has a more multidimensional feeling to it vs the linear sense of stage.
- **Rafts help us re-locate**: rafts describe activities that support growth. Again in piagetian terminology we might say activities that generate  "accommodation".
- **Lines vs Domains**: the "lines" terminology comes more from Integral/Wilber. domains used more in academia.
- **Maps and Compasses 🧭 **: in addition to a map, we want ways to **locate** ourselves on the map. Sometimes this can be done implicitly by the map because the map has *features* (rivers, mountains, cities etc) that we can use to reference ourselves against. In developmental terms these would be various psycho-social features -- behaviours, cognitive patterns, emotional dynamics etc -- that we can reference against. However, often we will want an explicit "locational mechanism": a "compass" (or GPS) if we pursue the map metaphor. For example, Lectica's work has been an effort to create a reliable metric or compass for location in hierarchical skills development.

### A starter list of domains: Wake up, clean up, grow up, show up

Classic integral list is 4 distinct domains: "wake up, clean up, grow up, show up".

### Can matrix these with personal, collective and societal dimensions

Could multiply these by the personal, collective, societal dimensions of these. (not a perfect cross product as societal and showing up overlap)

<img src="https://github.com/life-itself/community/assets/180658/49ffcad6-5307-4ef6-87ee-5bca9a137266" width="600" />

> [!note]
> Still not very clear what the distinctions are here. For example, what is personal vs collective vs societal waking up? What is personal vs collective growing up? (is collective growing up just the aggregate of personal growing up or something more. This connects with discussions over "we-space" practices etc. 

### Domains as a framework for a research program

For each domain we could imagine having 3 things:

- An initial survey
- A map - of more or less completeness and details
- A list of rafts - with their relevance and effectiveness (ie. which locations the rafts help you move to) 

<img src="https://github.com/life-itself/community/assets/180658/231e0bd0-5790-441d-acef-7ed30400fe2f" width="400" />

[Image Source](https://link.excalidraw.com/l/9u8crB2ZmUo/8F30hhvlTcl)

### Initial field survey

An initial field survey could look across domains to ask what maps are there, what practices, what evaluations so far etc. Outcomes would be:

- Rough list of domains
- Maps: a map of the territory for each domain
- Rafts: practices, environments etc

## Existing usage

- #591 - long detailed comment https://github.com/orgs/life-itself/discussions/591#discussioncomment-6167371
- #859 - comment https://github.com/orgs/life-itself/discussions/859#discussioncomment-7256735 - (consolidated into above)
- #839 - commenbt https://github.com/orgs/life-itself/discussions/839#discussioncomment-7173001 - re Aletheia

And we've been using this kind of terminology at Life Itself Institute and Labs at least for a few years. (Originally, the naming had an intentional partial allusion to the "raft" teaching in Buddhism: once you have reached the other shore you leave the raft - and the map - behind)

**Comment by baouroux** [comment]
Very helpful overview, thanks Rufus.

**Comment by rufuspollock** [comment]
Moved to blog https://lifeitself.org/blog/domains-maps-rafts
